src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessThreadTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Security;
+using System.IO;
+using Xunit;
+using System.Threading;
+using System.Linq;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class ProcessThreadTests : ProcessTestBase
+    {
+        [Fact]
+        public void TestCommonPriorityAndTimeProperties()
+        {
+            ProcessThreadCollection threadCollection = _process.Threads;
+            Assert.True(threadCollection.Count > 0);
+
+            ProcessThread thread = threadCollection[0];
+
+            if (ThreadState.Terminated != thread.ThreadState)
+            {
+                Assert.Equal(_process.BasePriority, thread.BasePriority);
+                Assert.True(thread.CurrentPriority >= 0, "Thread CurrentPriority");
```

Comment:
That's why I previously added this:
https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/Diagnostics/AssertWithCallerAttributes.cs
While you're working on the tests you modify the test project to include this file, then all of the asserts will compile using this class rather than going directly against xunit. (There might be a few things here and there you need to fix up if the project is using any Assert methods I couldn't massage exactly, but that should be quick.)